src/applications/vaos/tests/appointment-list/components/AppointmentsPage/index.cancel.unit.spec.jsx

```diff
@@ -404,7 +404,7 @@ describe('VAOS <AppointmentsPage> cancellation:', () => {
     expect(modal).to.contain.text('Something went wrong');
     expect(modal).to.contain.text('Cheyenne VA Medical Center');
     expect(modal).to.contain.text('2360 East Pershing Boulevard');
-    expect(modal).to.contain.text('Cheyenne, WY 82001-5356');
+    expect(modal).to.contain.text('Cheyenne, WyomingWY 82001-5356');
```

Comment:
I dislike this, and I'm open to other ideas about how to verify these addresses. Both the full name and the abbreviation are part of `textContent` (they're shown/hidden with css and aria attributes). JSDom doesn't support innerText, which in theory might only show what's visible. You could break this into separate expects, but that just hides the weird way the text content looks.

Reply:
Just curious, but what changed that the original test is not working?

Reply:
Agreed! Nothing comes to mind other than using regex which just makes things worse.